### ISSUE TITLE
[Feature Request]: Filters of the same type should not share state

### ISSUE THREAD

**Issue by Nocifer** [VueTorrent/VueTorrent#1996]
### Description

If I filter by "category 1" and "category 2", then there is no way to disable or remove one of those filters and keep the other one, because the only "sub-chip" I can interact with is "Category filter: 2 categories" which handles all the "Category" filters at once, so the only solution is to remove all of them and then add back the filter I wanted to keep.

The same thing happens with tags, etc.

### Describe the solution you'd like

Each filter should be a separate entry in the active filter list and it should be able to disable/remove each filter individually.

### Complementary informations

- [ ] Is this feature already implemented in the default WebUI?

**Comment by Larsluph** [comment]
This must be thought through and designed properly as to not overflow user informations he already know.

The current feature has been designed having that in mind, to prevent having dozens of chip spawning all at once.

**Comment by Nocifer** [comment]
I agree that the UX concerns are real, but unfortunately the current solution hampers usability, so it would probably be worthwhile to find a better compromise.

Perhaps the filter type sub-chips could further expand when hovered over/clicked upon and display the individual filters?

**Comment by Larsluph** [comment]
The idea I can come up to is to make a sub-list on the right of each chip to list all applied filters when hovering over a grouping chip.

**Comment by Nocifer** [comment]
Sorry for the very late reply. Yeah, I can see that working very nicely, as long as that sub-list is individually actionable, just like the grouping chips are actionable.

I.e. currently we can disable filter groupings either one by one by clicking on them, or all of them by clicking the "Active filters" chip; similarly, we should be able to disable individual categories in a grouping like "Category" by clicking on them in the sub-list, or disable any and all category filtering by clicking the "Category" grouping chip.

And yes, I know I'm stating the obvious here :)
